code-samples/creating-your-bot/command-handling/commands/utility/options-info.js

```diff
@@ -0,0 +1,13 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('options-info')
+		.setDescription('Information about the options provided.')
```

Comment:
Doesn't really provide any information, maybe echo would be a more applicable example?
Though I'm not sure what this is trying to demonstrate in the first place; never appears in the written guide either, unless I'm missing something

Reply:
I didn't understand this command either. Renaming this to echo would be more clear and wouldn't cause further confusion